src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -112,19 +112,26 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "beautifulsoup4": ("bs4",),
     "bitvector": ("BitVector",),
     "cattrs": ("cattr", "cattrs"),
+    "django-activity-stream": ("actstream",),
     "django-cors-headers": ("corsheaders",),
     "django-countries": ("django_countries",),
     "django-filter": ("django_filters",),
     "django-fsm": ("django_fsm",),
+    "django-notifications-hq": ("notifications",),
+    "django-oauth-toolkit": ("oauth2_provider",),
     "django-object-actions": ("django_object_actions",),
+    "django-otp-yubikey": ("otp_yubikey",),
```

Comment:
I think this one might be covered by the catch-all too, which takes everything after `django-`, replacing `-` with `_`?

```suggestion
```